lib/models/dockerfile.ts

```
@@ -17,6 +17,18 @@ export default class Dockerfile {
     return fs.readFileSync(this.path, "utf8");
   }
 
+  get dependencyFiles(): string[] {
```

```suggestion
  get dependencyFilePaths(): string[] {
```

(mirror the environment variable naming)